### PR TITLE
Feature: add ability to generate deposit address by admin

### PR DESCRIPTION
**Pull request by oyershov**

### CODE REVIEW

File: app/api/v2/admin/entities/deposit.rb [hunk 1]
```diff
@@ -23,6 +23,15 @@ class Deposit < API::V2::Entities::Deposit
             }
           )
 
+          expose(
```
Comment (mnaichuk) [inline app/api/v2/admin/entities/deposit.rb]: no need to add this we have it in the parent class

File: app/api/v2/admin/deposits.rb [hunk 2]
```diff
@@ -135,6 +135,38 @@ class Deposits < Grape::API
...
+                   desc: -> { API::V2::Admin::Entities::Deposit.documentation[:currency][:desc] }
+          given :currency_id do
+            optional :address_format,
+                    type: String,
+                    values: { value: -> { %w[legacy cash] }, message: 'admin.deposit.invalid_address_format' },
+                    validate_currency_address_format: { value: true, prefix: 'admin.deposit' },
+                    desc: 'Address format legacy/cash'
+          end
+        end
+        get '/deposit_address' do
```
Comment (mnaichuk) [inline app/api/v2/admin/deposits.rb]: better to use POST here. From API addresses returns from `/api/v2/peatio/admin/members/UID` endpoint. So here it should be like you send POST request to generate a deposit address for user and then you can get update from call that I sent before

File: app/api/v2/admin/deposits.rb [hunk 3]
```diff
@@ -135,6 +135,38 @@ class Deposits < Grape::API
             status 422
           end
         end
+
+        desc 'Returns deposit address for account you want to deposit to by currency.',
```
Comment (mnaichuk) [inline app/api/v2/admin/deposits.rb]: change title here

**Review by mnaichuk — CHANGES_REQUESTED** [review]

File: app/api/v2/admin/deposits.rb [hunk 4]
```diff
@@ -135,6 +135,40 @@ class Deposits < Grape::API
...
+          given :currency_id do
+            optional :address_format,
+                    type: String,
+                    values: { value: -> { %w[legacy cash] }, message: 'admin.deposit.invalid_address_format' },
+                    validate_currency_address_format: { value: true, prefix: 'admin.deposit' },
+                    desc: 'Address format legacy/cash'
+          end
+        end
+        post '/deposit_address' do
+          admin_authorize! :create, Deposit
```
Comment (mnaichuk) [inline app/api/v2/admin/deposits.rb]: : create, Payment Address

**Review by mnaichuk — CHANGES_REQUESTED** [review]

**Review by mnaichuk — APPROVED** [review]

**Comment by calj** [comment]
Please squash/rebase and final check

**Pull request merged**